js/views/comment-form.js

```diff
@@ -16,6 +19,14 @@ var Utils = require("../util/utils");
 
 var CHARACTER_LIMIT = constants.CHARACTER_LIMIT;
 
+window.onload = function() {
+  var comment = sessionStorage.getItem("comment");
```

Comment:
@urakagi Is this designed to make it so that when a user logs in with email and comes back to the page, they get the same comment they were last looking at? If so, I think there may be an issue with this approach, as this would only replace the comment text, but not the current comment id, which is presumably what actually gets used to figure out what update to send to the db when they vote. In other words, the way things are set up here, I'm concerned users could be submitting votes for comments other than the one they think they are submitting in this case.

Reply:
Yes, it is meant for that feature, but as you say it's not good. I will remove this.